p/rsag/example.js

```diff
@@ -11,7 +11,7 @@ const albertEinsteinStr = {
 	type: 'location',
 	id: '990004201',
 	address: 'Rostock - Südstadt, Albert-Einstein-Straße 23',
-	latitude: 54.077208, longitude: 12.108299
+	latitude: 54.077208, longitude: 12.108299,
```

Comment:
Actually this seems to be a good change. For example if someone will later add a line, the diff will be clearer. Do you also want to get rid of changes like this?

Reply:
It's indeed in line with what I explained in https://github.com/public-transport/hafas-client/pull/305#issuecomment-1837485331, let's keep it.